src/main/scala/com/danielasfregola/twitter4s/entities/GeoBoundingBoxes.scala

```diff
@@ -0,0 +1,11 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class GeoBoundingBoxes(boundingBoxes: Seq[GeoBoundingBox]) {
```

Comment:
Not sure we actually need this case class...do we? At the end this is just `Seq[GeoBoundingBox]`...

Reply:
I actually think that this could be completely removed by using Seq[GeoBoundingBox] instead in the methods

Reply:
I created this case class so that we don't need to do the conversion to query params, that is:

    boundingBoxes.flatMap(_.toLngLatPairs)

in each client method, e.g. in `TwitterStatusClient.filterStatuses`, I can just call `locations.toLngLatPairs`.

But, the bad thing is that we need to wrap it in another case class which looks redundant. What do you think?

Reply:
Having a case class just to have that method seems a bit too much to me too.
We could achieve the same by adding the function to the companion object for `GeoBoundingBox `. Something like the following:

```
object GeoBoundingBox {
    def toLngLatPairs(boundingBoxes: Seq[GeoBoundingBox]) = ???
}
```

and then ...

```
import GeoBoudingBox._
toLngLatPairs(boundingBoxes)
```

Reply:
Yeah, I agree.